sdp/processors/datasets/librispeech/create_initial_manifest.py

```diff
@@ -0,0 +1,137 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import fnmatch
+import glob
+import json
+import os
+import typing as tp
```

Comment:
Nitpick but: could you just leave it as `import typing` without the alias? Since we never use an alias in NeMo repo.

Reply:
done